rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -715,6 +715,22 @@ namespace vk
 				// See https://bugs.freedesktop.org/show_bug.cgi?id=110970
 				rsx_log.fatal("RADV drivers have a major driver bug with LLVM 8.0.0 resulting in no visual output. Upgrade to LLVM version 8.0.1 or greater to avoid this issue.");
 			}
+			else if (get_driver_vendor() == driver_vendor::NVIDIA)
```

Comment:
Not trying to be a shill but I'd argue that this code path is more likely to be executed than the one above

Reply:
It gets called once per game, there is no performance consideration here.